vMenu/menus/VehicleOptions.cs

```diff
@@ -897,6 +915,17 @@ private void CreateMenu()
                     {
                         Notify.Error(CommonErrors.NoVehicle);
                     }
+                } else if (item == radioStations)
+                {
+                    RadioStation newStation = (RadioStation)Enum.GetValues(typeof(RadioStation)).GetValue(listIndex);
```

Comment:
Since the list item has all the radio stations in order, you don't need to parse the index like that, you can just do `(RadioStation)listIndex`, pretty sure that should work fine.

Reply:
I think i do have to do this. RadioOff has a value of 255, and the listIndex is 21 (or 22 not sure).